src/cephadm/samples/rgw_ssl.json

```diff
@@ -0,0 +1,103 @@
+{
+  "rgw_realm": "default",
+  "rgw_zone": "default",
+  "service_type": "rgw",
+  "placement": {
+    "hosts": [{
+      "hostname": "ironic-moliver",
+      "name": "ironic-moliver",
+      "network": "ironic-moliver"
```

Comment:
ignore all this for now, will clean it up (or remove this sample) before we merge.